extendeddatasquare.go

```diff
@@ -109,18 +111,18 @@ func ImportExtendedDataSquare(
 // NewExtendedDataSquare returns a new extended data square with a width of
 // edsWidth. All shares are initialized to nil so that the returned extended
 // data square can be populated via subsequent SetCell invocations.
-func NewExtendedDataSquare(codec Codec, treeCreatorFn TreeConstructorFn, edsWidth uint, chunkSize uint) (*ExtendedDataSquare, error) {
+func NewExtendedDataSquare(codec Codec, treeCreatorFn TreeConstructorFn, edsWidth uint, shareSize uint) (*ExtendedDataSquare, error) {
```

Comment:
This looks like it's part of the public API, right? If so this is a breaking change.